src/isar/models/mission/mission.py

```diff
@@ -115,10 +115,10 @@ def set_unique_id_and_metadata(self) -> None:
 
     def _set_unique_id(self) -> None:
         plant_short_name: str = settings.PLANT_SHORT_NAME
-        robot_id: str = settings.ROBOT_ID
+        isar_id: str = settings.ISAR_ID
         now: datetime = datetime.utcnow()
         self.id = (
-            f"{plant_short_name.upper()}{robot_id.upper()}"
+            f"{plant_short_name.upper()}{isar_id.upper()}"
```

Comment:
This is one example where we should still use the robot name I think.

What this function does is create the unique identifier for the mission which is currently running. It is then used as a folder name etc. for uploads. Additional guarantees for uniqueness in this one is by using a datetime object as part of the name.

Reply:
So change `ISAR_ID` to `ROBOT_NAME` for this function.

Reply:
done